Source/WebKit/WebProcess/WebPage/Cocoa/WebPageCocoa.mm

```diff
@@ -670,12 +670,19 @@
 
     RefPtr enclosingFormControl = enclosingTextFormControl(selection.start());
     if (RefPtr editableRootOrFormControl = enclosingFormControl.get() ?: selection.rootEditableElement()) {
-        postLayoutData.editableRootIsTransparentOrFullyClipped = result.isContentEditable && isTransparentOrFullyClipped(*editableRootOrFormControl);
+        postLayoutData.selectionIsTransparentOrFullyClipped = result.isContentEditable && isTransparentOrFullyClipped(*editableRootOrFormControl);
 #if PLATFORM(IOS_FAMILY)
         result.visualData->editableRootBounds = rootViewInteractionBounds(Ref { *editableRootOrFormControl });
 #endif
+    } else {
+        auto posStart = selection.start();
+        auto posEnd = selection.end();
+
+        RefPtr ancestorContainer = Position::getCommonAncestorForPositions(posStart, posEnd);
+        postLayoutData.selectionIsTransparentOrFullyClipped = isTransparentOrFullyClipped(*ancestorContainer);
```

Comment:
Probably best to be robust in the case where there's no common ancestor (instead of crashing)

```suggestion
        if (RefPtr ancestorContainer = commonInclusiveAncestor(selection.start(), selection.end()))
            postLayoutData.selectionIsTransparentOrFullyClipped = isTransparentOrFullyClipped(*ancestorContainer);
```